src/esp/physics/ArticulatedObject.h

```diff
@@ -446,6 +461,40 @@ class ArticulatedObject : public esp::physics::PhysicsObjectBase {
     return objectIdToLinkId_;
   }
 
+  /**
+   * @brief Given the list of passed points in this object's local space, return
+   * those points transformed to world space.
+   * @param points vector of points in object local space
+   * @param linkID Unused for rigids.
+   * @return vector of points transformed into world space
+   */
+  std::vector<Mn::Vector3> transformLocalPointsToWorld(
+      const std::vector<Mn::Vector3>& points,
+      int linkID) const override {
+    auto linkIter = links_.find(linkID);
+    if (linkIter != links_.end()) {
+      return linkIter->second->transformLocalPointsToWorld(points, linkID);
+    }
+    return points;
```

Comment:
Silent failure if the passed link_id is not valid?

Reply:
Yes, currently. Do you have a preference for how to deal with this and below?

Reply:
I'll fail like we do everywhere else, with ESP_CHECK